src/AddIns/Uno.UI.MSAL/Uno.UI.MSAL.csproj

```diff
@@ -38,7 +38,7 @@
 	</ItemGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Microsoft.Identity.Client" Version="4.15.0" />
+		<PackageReference Include="Microsoft.Identity.Client" Version="4.47.0" />
```

Comment:
There are other `csproj` files in this directory where the version should be updated.

Reply:
There are references in SamplesApp as well (which is why the build fails)